notebooks/en/llm_judge_evaluating_ai_search_engines_with_judges_library.ipynb

```diff
@@ -0,0 +1,1687 @@
+{
```

Comment:
<p>"<a href="https://github.com/quotient-ai/judges" target="_blank">judges</a> is an open-source library to use and create LLM-as-a-Judge evaluators. It provides a set of curated, research-backed..."</p><p> </p><p>"...a collection of real-world Google queries...as our benchmark for comparing..."</p><p> </p><p>"...which only includes human evaluated answers and their corresponding queries for correctness, clarity, and completeness."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done!</p>